runtime/lua/vim/lsp.lua

```diff
@@ -775,6 +778,14 @@ function lsp.start_client(config)
       off = 'off'; messages = 'messages'; verbose = 'verbose';
     }
     local version = vim.version()
+
+    if not config.workspaceFolders then
```

Comment:
I think this should be `config.workspace_folders` to match our snake_case convention, and plural not singular.